content/src/archived-fundings/nj-covest-fund.md

```diff
@@ -39,9 +39,10 @@ The NJ CoVest Fund provides seed funding to New Jersey technology and life scien
 - Early-stage technology and life science businesses located in physical commercial office, co-working, or incubator space in New Jersey
 - At least 2 full-time founders that have a financial commitment to the company, and a minimum of 50% of its W-2 employees in New Jersey
 
-> **Benefits**
->
-> - Up to $250,000 is available
-> - The NJ CoVest Fund provides growth capital to support early-stage companies from product development to successful, scalable commercialization
-> - This funding is available at the critical stage between product development and commercial operation expansion, where limited funds are available and a funding gap exists
-> - Investments through the NJ CoVest Fund will help stimulate business growth, additional capital investment, and the creation of high-skilled jobs in New Jersey
+:::callout{ header="Benefits" icon="false" calloutType="conditional" }- Up to $250,000 is available
```

Comment:
[boulder] I tested this locally, if there is content on this line it will break the widget.
```suggestion
:::callout{ header="Benefits" icon="false" calloutType="conditional" }
- Up to $250,000 is available
```